test/acs-engine-test/main.go

```diff
@@ -150,6 +153,11 @@ func (m *TestManager) testRun(d config.Deployment, index, attempt int, timeout t
 		rgPrefix = "y"
 		fmt.Printf("RESOURCE_GROUP_PREFIX is not set. Using default '%s'\n", rgPrefix)
 	}
+	// Randomize region if no location was configured
+	if d.Location == "" {
+		randomIndex := rand.Intn(len(m.regions))
```

Comment:
Ok so I noticed locally that unless I did -

```
r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

I could not rely on the RNG to give me back a truly distributed random number. Not that you need to do that here but just something I noticed when generating random suffixs for resource groups.